cmd/registers/src/lib.rs

```diff
@@ -169,14 +185,14 @@ fn print_reg(reg: Register, val: u32, fields: &[RegisterField]) {
         }
     }
 
-    fn print_bars(f: &[RegisterField], elbow: bool) {
-        let mut pos = 32;
+    let print_bars = |f: &[RegisterField], elbow: bool| {
+        let mut pos = reg_size;
 
         for i in 0..f.len() {
             while pos > f[i].lowbit {
                 print!(" ");
 
-                if pos % 4 == 0 && pos != 32 {
+                if pos % 4 == 0 && pos != reg_size {
```

Comment:
Do you really mean `pos % 4`?

Reply:
Yea, we are just counting off the bits into chunks of 4.